ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -572,13 +594,20 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
     }
 
     renderBillingDescription() {
+      const {enableBillingUpgrade} = serverConfigStore.get().config;
       return <div>
         The <AouTitle/> provides $300 in free credits per user. Please refer to
         <StyledExternalLink href={supportUrls.billing} target='_blank'> &nbsp;this article
         </StyledExternalLink> to learn more about the free credit
-        program and how it can be used. Once you have used up your free credits, you can request
-        additional credits by <span style={styles.link} onClick={() => this.openContactWidget()}>
-        contacting support</span>.
+        program and how it can be used .
+        {!enableBillingUpgrade &&
+        <div style={{display: 'inline'}}>Once you have used up your free credits, you can request
+          additional credits by <span style={styles.link} onClick={() => this.openContactWidget()}>
+        contacting support</span>.</div>}
+        {enableBillingUpgrade &&
+        <div style={{display: 'inline'}}>Once you have used up your free credits, you can either select a shared billing account or create
+          a new one using either Google Cloud Platform or a Google billing partner.
+          Please note once a new billing account is created it may take a few days to show up in the <b>Select account</b> dropdown.</div>}
```

Comment:
A few days??? What is this talking about?

Reply:
I don't know, I copied from the UX, does it make sense, shall I check with Lou on this?
https://projects.invisionapp.com/d/main#/console/21577223/456332935/preview

Reply:
Yes, please check with Shimon or Sophie on this, I guess it might be referring to Carahsoft; in which case this wording needs to be adjusted.